### PR TITLE
Add missing logging of errors inside catch blocks

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/de/sub/goobi/forms/SessionForm.java [hunk 1]
```diff
@@ -37,6 +39,7 @@
     private SimpleDateFormat formatter = new SimpleDateFormat("HH:mm:ss");
     private String aktuelleZeit = this.formatter.format(new Date());
     private String bitteAusloggen = "";
+    private static final Logger logger = LogManager.getLogger(ModuleServerForm.class);
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/forms/SessionForm.java]: This class is called `SessionForm` but you are creating a logger for class `ModuleServerForm`

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java [hunk 2]
```diff
@@ -45,6 +47,7 @@
     List<DocStruct> docStructsOhneSeiten;
     Process myProcess;
     boolean autoSave = false;
+    private static final Logger logger = LogManager.getLogger(MetadatenHelper.class);
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java]: This class is called `MetadatenVerifizierung` but you are creating a logger for class `MetadatenHelper`.

File: Kitodo/src/main/java/de/sub/goobi/forms/SessionForm.java (hunk 1 above)
Comment (Beacze) [inline Kitodo/src/main/java/de/sub/goobi/forms/SessionForm.java]: fixed

File: Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java (hunk 2 above)
Comment (Beacze) [inline Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenVerifizierung.java]: fixed

**Pull request merged**